app/src/main/java/com/isc/hermes/controller/MapWayPointController.java

```diff
@@ -21,6 +34,7 @@ public class MapWayPointController implements MapClickConfigurationController {
     private final WaypointOptionsController waypointOptionsController;
     private boolean isMarked;
     private Context context;
+    private int markerCount = 0;
```

Comment:
markerCount should be initialized in the constructor

Reply:
Done